Userland/Libraries/LibJS/Runtime/Completion.h

```diff
@@ -15,6 +15,18 @@
 
 namespace JS {
 
+#define TRY_OR_THROW_OOM(vm, expression)                                       \
```

Comment:
There's already `TRY_OR_RETURN_OOM` in LibWeb, and having both without any scoping makes me a bit uncomfortable. Usually we could just use `JS_` and `WEB_` prefixes, doesn't seem ideal for `TRY` macros though. Not sure what the best solution is...

Reply:
Hmm I wonder if we can get by with only have the LibJS version of this macro. LibWeb exceptions can wrap around LibJS throw completions, right?

Reply:
Yeah this actually worked great. So the commit to add the macro to LibJS is now a commit to move the macro from LibWeb to LibJS.

Reply:
Yeah this actually worked great. So the commit to add the macro to LibJS is now a commit to move the macro from LibWeb to LibJS.

Reply:
Nice!